File: template/go/template.go
package main

import (
	"bufio"
	"fmt"
	"math"
	"os"
	"strconv"
	"strings"
)

/**
 * Author       :   %s
 * Date         :   %s
 * ver.         :   %s
 * link         :   %s
 * dir          :   %s
 */

const MOD = 1_000_000_007 // 10^9+7

var (
	reader *bufio.Reader
	writer *bufio.Writer
)

func init() {
	reader = bufio.NewReader(os.Stdin)
	writer = bufio.NewWriter(os.Stdout)
}

func code(a []int, n int) int {
	return 0
}

func main() {
	defer writer.Flush()

	// const TOTAL_TEST_CASES = 1
	TOTAL_TEST_CASES := nextInt()

	for testCase := 0; testCase < TOTAL_TEST_CASES; testCase++ {
		// Taking input
		// n := nextInt()
		// s := next()
		// a := inputArray(n)

		// Compute
		// res := code(a, n)
		// fmt.Fprintf(writer, "DEBUG: Case #%d: %d\n", testCase+1, res)

		// Storing the result
		// fmt.Fprintln(writer, res)
	}
}

// ==================== Input Helper Functions ====================

func next() string {
	var token string
	fmt.Fscan(reader, &token)
	return token
}

func nextInt() int {
	var n int
	fmt.Fscan(reader, &n)
	return n
}

func nextInt64() int64 {
	var n int64
	fmt.Fscan(reader, &n)
	return n
}

func nextFloat64() float64 {
	var f float64
	fmt.Fscan(reader, &f)
	return f
}

func nextLine() string {
	line, _ := reader.ReadString('\n')
    return strings.TrimRight(line, "\r\n")
}

func inputArray(n int) []int {
	a := make([]int, n)
	for i := 0; i < n; i++ {
		a[i] = nextInt()
	}
	return a
}

func inputArray64(n int) []int64 {
	a := make([]int64, n)
	for i := 0; i < n; i++ {
		a[i] = nextInt64()
	}
	return a
}

func inputMatrix(r, c int) [][]int {
	matrix := make([][]int, r)
	for i := 0; i < r; i++ {
		matrix[i] = inputArray(c)
	}
	return matrix
}

// ==================== Utility Functions ====================

func displayMatrix(matrix [][]int) {
	for _, row := range matrix {
		fmt.Println(row)
	}
}

func gcd(a, b int) int {
	if b == 0 {
		return a
	}
	return gcd(b, a%b)
}

func lcm(a, b int) int {
	return (a * b) / gcd(a, b)
}

func isPrime(n int) bool {
	if n <= 1 {
		return false
	}
	if n <= 3 {
		return true
	}
	if n%2 == 0 || n%3 == 0 {
		return false
	}
	for i := 5; i*i <= n; i += 6 {
		if n%i == 0 || n%(i+2) == 0 {
			return false
		}
	}
	return true
}

func abs(x int) int {
	if x < 0 {
		return -x
	}
	return x
}

func min(a, b int) int {
	if a < b {
		return a
	}
	return b
}

func max(a, b int) int {
	if a > b {
		return a
	}
	return b
}

func pow(base, exp int) int {
	result := 1
	for exp > 0 {
		if exp%2 == 1 {
			result *= base
		}
		base *= base
		exp /= 2
	}
	return result
}

func modPow(base, exp, mod int) int {
	result := 1
	base %= mod
	for exp > 0 {
		if exp%2 == 1 {
			result = (result * base) % mod
		}
		base = (base * base) % mod
		exp /= 2
	}
	return result
}

func sum(arr []int) int {
	total := 0
	for _, v := range arr {
		total += v
	}
	return total
}

func reverseArray(arr []int) {
	for i, j := 0, len(arr)-1; i < j; i, j = i+1, j-1 {
		arr[i], arr[j] = arr[j], arr[i]
	}
}

// ==================== String Conversion ====================

func toInt(s string) int {
	n, _ := strconv.Atoi(s)
	return n
}

func toInt64(s string) int64 {
	n, _ := strconv.ParseInt(s, 10, 64)
	return n
}

func toString(n int) string {
	return strconv.Itoa(n)
}

// ==================== Math Helpers ====================

func sqrtInt(n int) int {
	return int(math.Sqrt(float64(n)))
}

func ceilDiv(a, b int) int {
	return (a + b - 1) / b
}

File: template/java/template.java
import java.io.*;
import java.util.*;
import java.math.*;
import java.util.stream.*;

/**
 * Author       :   %s
 * Date         :   %s
 * ver.         :   %s
 * link         :   %s
 * dir          :   %s
 */

// public class OBrutusSolution {
class OBrutusSolution {
    private static final OBrutusSolutionTemplate sc = new OBrutusSolutionTemplate();
    private static final StringBuilder sb = new StringBuilder();

    public static int code(int[] a, int n) {
        return 0;
    }

    public static void main(String[] args) {
        // 64MB stack so deep DFS/recursion won't StackOverflow on big inputs.
        new Thread(null, OBrutusSolution::run, "main", 1 << 26).start();
    }

    private static void run() {
        // final int TOTAL_TEST_CASES = 1;
        final int TOTAL_TEST_CASES = sc.nextInt();

        for (int testCase = 1; testCase <= TOTAL_TEST_CASES; testCase++) {
            // Taking input
            // int n = sc.nextInt();
            // String s = sc.next();
            // int[] a = sc.input(n);

            // Compute
            var res = code(a, n);
            // System.out.println("DEBUG: Case #" + testCase + ": " + res);

            // Storing the result — chained append, NOT (res + "\n"): no per-line String garbage.
            sb.append(res).append('\n');
        }

        // Single bulk write of all output.
        try {
            System.out.write(sb.toString().getBytes());
            System.out.flush();
        } catch (IOException e) {
            throw new UncheckedIOException(e);
        }
    }
}

class OBrutusSolutionTemplate {
    private final InputStream in;
    private final byte[] buf = new byte[1 << 16];
    private int ptr = 0, len = 0;
    public static final int MOD = 1_000_000_007; // 10^9+7

    OBrutusSolutionTemplate() {
        this.in = System.in;
    }

    OBrutusSolutionTemplate(String input) {
        InputStream s;
        try {
            s = new FileInputStream(input);
        } catch (IOException e) {
            e.printStackTrace();
            s = System.in;
        }
        this.in = s;
    }

    private int read() {
        if (ptr == len) {
            try {
                len = in.read(buf, 0, buf.length);
            } catch (IOException e) {
                throw new UncheckedIOException(e);
            }
            ptr = 0;
            if (len <= 0) return -1;
        }
        return buf[ptr++];
    }

    int nextInt() {
        int c = read();
        while (c <= ' ') {
            if (c == -1) throw new RuntimeException("EOF");
            c = read();
        }
        boolean neg = c == '-';
        if (neg) c = read();
        int x = 0;
        while (c >= '0' && c <= '9') {
            x = x * 10 + (c - '0');
            c = read();
        }
        return neg ? -x : x;
    }

    long nextLong() {
        int c = read();
        while (c <= ' ') {
            if (c == -1) throw new RuntimeException("EOF");
            c = read();
        }
        boolean neg = c == '-';
        if (neg) c = read();
        long x = 0;
        while (c >= '0' && c <= '9') {
            x = x * 10 + (c - '0');
            c = read();
        }
        return neg ? -x : x;
    }

    String next() {
        int c = read();
        while (c <= ' ') {
            if (c == -1) return null;
            c = read();
        }
        StringBuilder s = new StringBuilder();
        while (c > ' ') {
            s.append((char) c);
            c = read();
        }
        return s.toString();
    }

    double nextDouble() {
        return Double.parseDouble(next());
    }

    String nextLine() {
        StringBuilder s = new StringBuilder();
        int c = read();
        while (c != '\n' && c != -1) {
            if (c != '\r') s.append((char) c);
            c = read();
        }
        return (s.length() == 0 && c == -1) ? null : s.toString();
    }

    public int[] input(int n) {
        int[] a = new int[n];
        for (int i = 0; i < n; i++)
            a[i] = nextInt();
        return a;
    }

    public int[] input(int[] a, int n) {
        for (int i = 0; i < n; i++)
            a[i] = nextInt();
        return a;
    }

    public long[] inputLong(int n) {
        long[] a = new long[n];
        for (int i = 0; i < n; i++)
            a[i] = nextLong();
        return a;
    }

    public int[][] input(int r, int c) {
        int[][] a = new int[r][c];
        for (int[] x : a)
            input(x, c);
        return a;
    }

    public void displayMatrix(int[][] a) {
        for (int[] x : a)
            System.out.println(Arrays.toString(x));
    }

    public long gcd(long a, long b) {
        while (b != 0) {
            long t = a % b;
            a = b;
            b = t;
        }
        return a;
    }

    public boolean isPrime(int n) {
        if (n < 2) return false;
        if (n % 2 == 0) return n == 2;
        if (n % 3 == 0) return n == 3;
        for (int i = 5; (long) i * i <= n; i += 6)
            if (n % i == 0 || n % (i + 2) == 0) return false;
        return true;
    }
}
